db/sg_replicate_cfg.go

```diff
@@ -434,6 +468,12 @@ func (m *sgReplicateManager) InitializeReplication(config *ReplicationCfg) (repl
 		return nil, err
 	}
 
+	// Explicitly defined auth credentials of remote database takes precedence over
+	// the credentials specified in the database URL.
```

Comment:
Should we return an error in `ValidateReplication` if both are set?

Reply:
Yeah, that's a good idea. Added changes in ValidateReplication to throw appropriate validation error when both are set.